charts/zabbix/secret.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+data:
+  dbname: emFiYml4
+  host: emJ4LXphYmJpeC1wb3N0Z3Jlc3Fs
+  password: cVZha1dydEdHSktKT0ZQRwo=
+  port: NTQzMg==
+  user: emFiYml4
```

Comment:
Hi @sa-ChristianAnton!
Can this values be dynamics like .values.PARAMETER_NAME ?